spec/support/schema.graphql

```diff
@@ -151,6 +157,30 @@ interface DirectoryEntry {
   path: String!
 }
 
+# A Document is a container for OMS
+interface Document {
+  # All DocumentLinks that this Document is part of
+  documentLinks: [DocumentLink!]!
```

Comment:
Perhaps merge with `documentLinksBySource` and `documentLinksByTarget` and specify direction by argument `origin` with values `any` (default), `source` or `target`

Reply:
Done with 940aec11